cmd/network-tracer/tracer.go

```diff
@@ -63,8 +64,22 @@ func (nt *NetworkTracer) Run() {
 	http.HandleFunc("/status", func(w http.ResponseWriter, req *http.Request) {})
 
 	http.HandleFunc("/connections", func(w http.ResponseWriter, req *http.Request) {
-		cs, err := nt.tracer.GetActiveConnections()
 
+		// We require to send a client_id to the network state
+		rawCID := req.URL.Query().Get("client_id")
+		clientID := ebpf.DEBUGCLIENT
+
+		if rawCID != "" {
+			var err error
+			clientID, err = strconv.Atoi(rawCID)
+			if err != nil {
+				log.Errorf("wrong clientID: '%s': %s", rawCID, err)
```

Comment:
Why not just let `clientId` be a string to completely remove the error potential of this?